Ginger/GingerCoreCommon/Repository/BusinessFlowLib/Activity.cs

```diff
@@ -59,9 +59,9 @@ public enum eItemParts
     }
     public enum eHandlerMappingType
     {
+        None = 0,
```

Comment:
i think changing the enum default will break existing flows ?

To verify open any existing business flow, and the activity error handler mapping should be set to All available error handler by default